pkgs/applications/audio/wolf-spectrum/default.nix

```
@@ -0,0 +1,50 @@
+{ pkgs
```

avoid importing pkgs namespace
```suggestion
{ stdenv
```